libarchive/test/test_sparse_basic.c

```diff
@@ -67,13 +67,13 @@ struct sparse {
 
 static void create_sparse_file(const char *, const struct sparse *);
 
-#if defined(__APPLE__)
-/* On APFS holes need to be at least 4096x4097 bytes */
-#define MIN_HOLE 16781312
-#else
-/* Elsewhere we work with 4096*10 bytes */
-#define MIN_HOLE 409600
-#endif
+/* This should be large enough that any OS/filesystem that
+ * does support sparse files is certain to store a gap this big
+ * as a hole. */
+/* A few data points:
+ * ZFS on FreeBSD needs this to be at least 200kB */
+/* macOS APFS needs this to be at 4096x4097 bytes */
```

Comment:
This comment is out of date now

Reply:
It is?   I'm setting the minimum hole size to 32MiB in order to be certain that it's larger than either of these values.

Reply:
Oh.  I see.  Yeah, it appears that APFS has a higher limit now, but I don't know exactly what it is.

Reply:
Ah, I get it now with the clarification. I was expecting to see the minimum documented value for macOS to go up to 32MiB.